### DISCUSSION TITLE
Multiple instances on MacOS

### DISCUSSION THREAD

**Discussion by ramiel** [tauri-apps/tauri#6650] [Q&A]
I'd like to run multiple instances of the same app on macOS (and all other OS actually).
This can also mean to be able to create a new window from the main menu. I read all the documentation but it's not clear how I can create a new window with the main content from the menu

**Comment by FabianLars** [comment]
I'm a bit confused, do you want actual multiple _separate_ **app** instances or multiple **windows** using the same app/backend?

Honestly no idea how to spawn multiple instances of an app on macos (from code), but spawning windows should be relatively straight forward. Since menu events can only be handled in Rust, you'll have to use the [`WindowBuilder`](https://docs.rs/tauri/latest/tauri/window/struct.WindowBuilder.html) to create windows. It could look something like this: https://github.com/tauri-apps/tauri/blob/b02fc90f450ff9e9d8a35ee55dc1beced4957869/examples/api/src-tauri/src/main.rs#L134-L139 (this is for system trays but it should be really similar for window menus)
